src/actions/events.js

```diff
@@ -1,15 +1,74 @@
+import database from "../firebase/firebase";
+
+export const setEvents = events => ({
+  type: "SET_EVENTS",
+  events,
+});
+
+export const startSetEvents = () => (dispatch, getState) => {
+  const { uid } = getState().auth;
```

Comment:
You have access to `uid` in place where you are using that action. Why just don't pass that `uid` as argument here instead of getting it from the state?